config.yaml

```diff
@@ -38,15 +38,20 @@ params:
   mash:
     sketch_size: 1000
   kraken:
-    kraken_db: test_data/kraken_test_db
-    bracken_db: test_data/kraken_test_db/75mers_distr.txt
-    # kraken_db: /home/qiz173/Works/WoL/profiling/ref107/kraken
-    # bracken_db: /home/qiz173/Works/WoL/profiling/ref107/kraken/75mers_distr.txt
+    # db: test_data/kraken_test_db
+    # kmers: test_data/kraken_test_db/75mers_distr.txt
+    db: /home/qiz173/Works/WoL/profiling/ref107/kraken
+    kmers: /home/qiz173/Works/WoL/profiling/ref107/kraken/75mers_distr.txt
```

Comment:
We want to leave out barnacle file paths on the repo

Reply:
These are the test databases. I created a mini Kraken/Bracken database and added to the repo (see the commented lines above). Shall I uncomment them?

Reply:
Yes, include the test databases here but delete the Barnacle ones.